drivers/dma/dma_stm32.c

```diff
@@ -780,6 +780,7 @@ static void dma_stm32_config_irq_0(const struct device *dev)
 	ARG_UNUSED(dev);
 
 	DMA_STM32_IRQ_CONNECT(0, 0);
+#if !defined(CONFIG_SOC_SERIES_STM32WB0X) /* Only 1 IRQ on STM32WB0 */
```

Comment:
Can't use `#if DT_INST_IRQ_HAS_IDX(0, 1)` ?

Reply:
Indeed, it should work. Changing.